deps/build.jl

```diff
@@ -79,10 +79,9 @@ init_deps()
 end
 
 if is_apple()
-    if Pkg.installed("Homebrew") === nothing
-        error("Homebrew package not installed, please run Pkg.add(\"Homebrew\")")
-    end
     using Homebrew
+    Homebrew.update()
+    Homebrew.add("imagemagick")
```

Comment:
i know.  however, some OS X's already have libmagicwand in /usr/local/lib, in which case BinDeps will choose that one and *not* install it with Homebrew.  this results in "delegate not found" errors.

for reasons i don't understand, the Homebrew libmagickwand is chosen if they both exist.  so the easy fix was just to force the installation.

is there a way to tell BinDeps *not* to use the one in /usr/local/lib?